tokio/src/runtime/scheduler/current_thread.rs

```diff
@@ -250,7 +252,20 @@ impl fmt::Debug for CurrentThread {
 // ===== impl Core =====
 
 impl Core {
-    fn pop_task(&mut self, handle: &Handle) -> Option<task::Notified<Arc<Handle>>> {
```

Comment:
Renamed `pop_task` -> `next_task_*` to match names in the multi-threaded scheduler.